playground/nextjs/pages/survey.tsx

```diff
@@ -0,0 +1,37 @@
+import { usePostHog } from 'posthog-js/react'
+import { useEffect, useState } from 'react'
+
+export default function Survey() {
+    const posthog = usePostHog()
+    const [surveys, setSurveys] = useState([] as unknown as Survey[])
```

Comment:
FYI, when I try to build this locally, I get a failure:

```
./pages/survey.tsx:6:61
Type error: 'Survey' refers to a value, but is being used as a type here. Did you mean 'typeof Survey'?

  4 | export default function Survey() {
  5 |     const posthog = usePostHog()
> 6 |     const [surveys, setSurveys] = useState([] as unknown as Survey[])
    |                                                             ^
  7 |     const [selectedSurvey, setSelectedSurvey] = useState('0190bc7b-7096-0000-126d-1e5e7021a80e')
  8 |     const handleChange = (event) => {
  9 |         setSelectedSurvey(event.target.value)
 
```

because I don't think the playground has access to these types.

Reply:
fixed